Userland/Libraries/LibTextCodec/ShiftJISConverter.h

```diff
@@ -0,0 +1,7064 @@
+/*
+ * Copyright (c) 2021, Łukasz Patron <priv.luk@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+
+namespace {
```

Comment:
Missing namespace name, did you mean `namespace TextCodec`?

Reply:
Done.